mason/site/header/head.mas

```diff
@@ -18,9 +23,13 @@
   <!-- <link rel="stylesheet" type="text/css" href="/css/dataTables.bootstrap.min.css"> -->
   <link rel="stylesheet" type="text/css" href="/css/buttons/buttons.bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="/css/bootstrap3-xl-class.css">
-  
+
+  <link rel="stylesheet" href="/css/bootstrap-responsive.min.css">
```

Comment:
could you use the assets plugin to load these css files only on your page? that way there is no chance that these css classes interfere with other pages